pyproject.toml

```diff
@@ -55,7 +55,9 @@ packages = ["eds_scikit"]
 version = {attr = "eds_scikit.__version__"}
 
 [project.urls]
-source = "https://github.com/aphp/EDS-Scikit"
```

Comment:
Homepage should be the same as documentation in our case, and Source / Repository should be defined. Cf https://peps.python.org/pep-0621/#example

Reply:
I used these keywords to display those links nicely in PyPI (see [here](https://github.com/pypi/warehouse/blob/01f87cfdb6275a8c8bc11b838618df246fc10fec/warehouse/templates/packaging/detail.html#L22))

Reply:
I still think "Source Code" is needed

https://github.com/scikit-learn/scikit-learn/blob/07958e66a146e015bd738c9c7e3be41ef5ab6097/setup.py#L45

https://pypi.org/project/scikit-learn/

Reply:
We have the "Download File" button just above, I think I'll do the job !